logscan.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from prefect import flow, task, get_run_logger
+from tiled.client import from_profile
+
+
+tiled_client = from_profile("nsls2")["srx"]
+tiled_client_raw = tiled_client["raw"]
+
+
+def find_scanid(logfile_path, scanid):
+    is_scanid = False
+    with open(logfile_path) as lf:
+        lines = lf.readlines()
+        for line in lines:
```

Comment:
```suggestion
for line in lf:
```

This way you don't have to load the entire file every time

Reply:
Added review to the code. Thanks for the suggestion!